common/persistence/cassandraPersistence.go

```diff
@@ -1056,16 +1056,6 @@ func (d *cassandraPersistence) UpdateWorkflowExecution(request *UpdateWorkflowEx
 		d.CreateWorkflowExecutionWithinBatch(startReq, batch, cqlNowTimestamp)
 		d.createTransferTasks(batch, startReq.TransferTasks, startReq.DomainID, *startReq.Execution.WorkflowId,
 			*startReq.Execution.RunId, cqlNowTimestamp)
-	} else if request.CloseExecution {
```

Comment:
are we going to keep this forever?

Reply:
yes, because the start workflow API need to know whether there is a previous run, if there is a prev run, what is the close status

Reply:
I think it is reasonable to clean it up after its retention days.

Reply:
ok let us have a discussion